test/claim-service.test.js

```diff
@@ -0,0 +1,73 @@
+
+describe('Test claim service', () => {
+  let claimService
+  let claimRepositoryMock
+  let minetypeRepositoryMock
+  let messageServiceMock
+
+  beforeEach(async () => {
+    claimRepositoryMock = require('../server/repository/claim-repository')
+    minetypeRepositoryMock = require('../server/repository/minetype-repository')
+    messageServiceMock = require('../server/services/message-service')
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')
+
+    claimService = require('../server/services/claim-service')

```

Comment:
Move the `require` to the top of the file.

Reply:
As claim service is not a mock, it will be cached by node so requiring it again before each test doesn't achieve anything.